### DISCUSSION TITLE
How to change signature help UI

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#1709] [General]
When using gK or space + K for signature help, I get a nice little popup explaining the code.

The problem is that the background color applied is almost identical to the background color of my terminal. 

Sometimes, you hardly notice that signature help is enabled.

![Screenshot 2023-10-14 at 01 18 38](https://github.com/LazyVim/LazyVim/assets/59118341/6463290d-cb82-4181-a2d6-a2543fec39b2)

How do I change the layout/color of the pop-up for signature help?

**Comment by dpetka2001** [comment]
[noice](https://github.com/folke/noice.nvim) is responsible for this. Check out its options. I believe it's the options in this [section](https://github.com/folke/noice.nvim#nui-options), but I'm not sure because I haven't tried it myself.
